File: src/main/java/at/scheinecker/intellij/coco/CocoCommenter.java
package at.scheinecker.intellij.coco;

import com.intellij.lang.Commenter;
import org.jetbrains.annotations.Nullable;

/**
 * Created by Thomas on 27/03/2015.
 */
public class CocoCommenter implements Commenter{
    @Nullable
    @Override
    public String getLineCommentPrefix() {
        return "//";
    }

    @Nullable
    @Override
    public String getBlockCommentPrefix() {
        return "/*";
    }

    @Nullable
    @Override
    public String getBlockCommentSuffix() {
        return "*/";
    }

    @Nullable
    @Override
    public String getCommentedBlockCommentPrefix() {
        return null;
    }

    @Nullable
    @Override
    public String getCommentedBlockCommentSuffix() {
        return null;
    }
}

File: src/main/kotlin/me/salzinger/intellij/coco/CocoCommenter.kt
package me.salzinger.intellij.coco

import com.intellij.lang.Commenter

/**
 * Created by Thomas on 27/03/2015.
 */
class CocoCommenter : Commenter {
    override fun getLineCommentPrefix(): String? {
        return "//"
    }

    override fun getBlockCommentPrefix(): String? {
        return "/*"
    }

    override fun getBlockCommentSuffix(): String? {
        return "*/"
    }

    override fun getCommentedBlockCommentPrefix(): String? {
        return null
    }

    override fun getCommentedBlockCommentSuffix(): String? {
        return null
    }
}
